README.md

```diff
@@ -44,3 +44,22 @@ A set of NPM Workspace commands are provided, and can be run from the root folde
 `npm run dev-lib`: Watch for changes in `@klimadao/lib` and recompile on save. Alternatively, run `npm run build-lib` to compile once (this is already done automatically for the above two commands).
 
 `npm run format-all`: Format all files with `prettier`.
+
+## Translatiing
+
+### For translators
+
+#### Setup
+
+- Create an account on [translation.io](https://translation.io)
+- Request access to the project atmosfearful/klimadao in the #content-translation KlimaDaO discord server
+
+#### Translating
```

Comment:
So when we run `sync` does it create changes in Git? For each locale except english right?

Or would you rather we only commit English and then just pull everything from the translation.io cloud at build time?

Tradeoffs for both approaches...

Reply:
So sync:
- pushes only message ids to translation.io
- pulls all translations in local po files
- 
It has nothing to do with git.
We should add a command after sync that does something like
```git add site/locale
git add app/locale
git commit -m "fetched changes from translation.io`
git push```